### PR TITLE
Issue #SC-1213 system settings api details added

### PR DESCRIPTION
**Pull request by Hari-stackroute**
System-settings related api details were added

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://433.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://433.qa.docs.sunbird.org/2.0.0/apis/systemsettingsapi// <br /> - http://433.qa.docs.sunbird.org/2.0.0/apis/systemsettingsapi// <br />  <br/> **Modified sections are -** <br />

File: apis/systemsettingsapi/systemsettings.yaml [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+          description: 'BAD REQUEST. The "list badges" operation failed ! The possible reason for failure is that you may have missed providing input for a mandatory parameter.'
+          schema:
+            $ref: '#/definitions/ContentCreateErrorResponse'
+        '500':
+          description: 'INTERNAL SERVER ERROR ! Looks like something went wrong! These errors are tracked automatically, but if the problem persists feel free to contact us. In the meantime, try refreshing.'
+  /get/{System_Setting_Id}:
+    get:
+      description:  >-
+       This API is associated with listing the system settings on the Sunbird Platform.
+          - The **_/get/{System_Setting_Id}_** endpoint executes the "DataV1SystemSettingsGetById" request based on parameters provided as metadata in the request body
```
Comment (manzarul) [inline apis/systemsettingsapi/systemsettings.yaml]: DataV1SystemSettingsGetById put relative name
GetSystemSettingById 
please look all other places as well.

**Comment by lakhanmandloi** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://433.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://433.qa.docs.sunbird.org/2.0.0/apis/systemsettingsapi// <br /> - http://433.qa.docs.sunbird.org/2.0.0/apis/systemsettingsapi// <br />  <br/> **Modified sections are -** <br />

File: apis/systemsettingsapi/systemsettings.yaml [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+  /list:
+    get:
+      description:  >-
+       This API is associated with listing the system settings on the Sunbird Platform.
+          - The **_/list_** endpoint executes the "DataV1SystemSettingsListGet" request based on parameters provided as metadata in the request body
+          - Required fields cannot be left null or empty
+      summary: Lists out the system settings
+      tags:
+      - System Settings APIs
+      operationId: DataV1SystemSettingsListGet
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply apis/systemsettingsapi/systemsettings.yaml]: changed to GetSystemSettingsList

File: apis/systemsettingsapi/systemsettings.yaml [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+      parameters:
+      - name: Content-Type
+        in: header
+        required: true
+        type: string
+        description: >-
+           The Content Type entity is the media type of the resource.
+            Possible media types can be:- 
+              - Application/json
+              - Multipart/form-data
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply apis/systemsettingsapi/systemsettings.yaml]: removed in all places

File: apis/systemsettingsapi/systemsettings.yaml [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+        type: string
+        description: 'The token/key used to execute the API'
+      - name: Authorization
+        in: header
+        required: true
+        type: string
+        description: 'All System-settings APIs require authorization for use. Specify the authorization key received from the administrator when placing the request for use of the API'
+      responses:
+        '200':
+          description: 'OK ! Successful operation."list badges" operation was successfully executed.'
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply apis/systemsettingsapi/systemsettings.yaml]: done

File: apis/systemsettingsapi/systemsettings.yaml [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+          schema:
+            $ref: '#/definitions/ContentCreateResponse'
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply apis/systemsettingsapi/systemsettings.yaml]: changed to SystemSettingsListResponse

File: apis/systemsettingsapi/systemsettings.yaml [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+        '400':
+          description: 'BAD REQUEST. The "list badges" operation failed ! The possible reason for failure is that you may have missed providing input for a mandatory parameter.'
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply apis/systemsettingsapi/systemsettings.yaml]: done

File: apis/systemsettingsapi/systemsettings.yaml [hunk 7]
```diff
@@ -0,0 +1,312 @@
...
+          schema:
+            $ref: '#/definitions/ContentCreateErrorResponse'
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply apis/systemsettingsapi/systemsettings.yaml]: changed to SystemSettingsErrorResponse, this is common error response for all api's

File: apis/systemsettingsapi/systemsettings.yaml (hunk 1 above)
Comment (Hari-stackroute) [reply apis/systemsettingsapi/systemsettings.yaml]: done

**Pull request merged**